src/routes/search-result-common.js

```diff
@@ -42,7 +42,7 @@ export const createRouteConfig = (searchTermValidator, requestPath, requestHandl
               return h.redirect(paths.SEARCH).takeover()
             }
 
-            metricsCounter(`search.${match.key}`)
+            metricsCounter(match.metricName)
```

Comment:
update to use the new `getMetricNameBySearchType` function as per the below comment(s)